shared/menubar/remote-proxy.desktop.tsx

```diff
@@ -67,23 +67,36 @@ const RemoteProxy = () => {
   const notifications = Container.useSelector(s => s.notifications)
   const {desktopAppBadgeCount, navBadges, widgetBadge} = notifications
 
-  const config = Container.useSelector(s => s.config)
-  const {daemonHandshakeState, loggedIn, outOfDate, username, windowShownCount} = config
-  const {httpSrvAddress, httpSrvToken} = config
-  const {avatarRefreshCounter: _arc, followers: _followers, following: _following} = config
-
-  const fs = Container.useSelector(s => s.fs)
-  const {pathItems, tlfUpdates, uploads, overallSyncStatus, kbfsDaemonStatus, sfmi} = fs
-
-  const chat2 = Container.useSelector(s => s.chat2)
-  const {inboxLayout, metaMap, badgeMap, unreadMap, participantMap} = chat2
+  const daemonHandshakeState = Container.useSelector(s => s.config.daemonHandshakeState)
```

Comment:
how this works could be better but its not worth this change right now. for now just pull out individual items so at least its limited to something this thing cares about